hack/update-mocks.sh

```diff
@@ -4,12 +4,33 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-CURR_DIR=$(pwd)
+source $(dirname $0)/library.sh
 
-for m in $*; do
+header "updating mock files"
+
+if [ ! -d "${REPO_ROOT}/vendor" ]; then
+  go mod vendor
+fi
+
+cd ${REPO_ROOT}
+if [ ! -e "${GOPATH}/bin/mockery" ]; then
+  ./hack/recurl.sh dist/mockery.tar.gz https://github.com/vektra/mockery/releases/download/v1.1.1/mockery_1.1.1_$(uname -s)_$(uname -m).tar.gz
+  tar zxvf dist/mockery.tar.gz mockery
+  chmod +x mockery
+  mkdir -p ${GOPATH}/bin
+  mv mockery ${GOPATH}/bin/mockery
+fi
+
+MOCKERY_CMD="${GOPATH}/bin/mockery"
+${MOCKERY_CMD} -version
+
+MOCK_FILES=$(find persist workflow -maxdepth 4 -not -path '/vendor/*' -not -path './ui/*' -path '*/mocks/*' -type f -name '*.go')
```

Comment:
This is tricky, to generate a mock file, it needs to be exiting in a proper directory first.